cmd/cli/kubectl-kyverno/report/report.go

```diff
@@ -54,6 +57,23 @@ func ComputePolicyReportResult(auditWarn bool, engineResponse engineapi.EngineRe
 	result.Message = ruleResponse.Message()
 	result.Source = kyverno.ValueKyvernoApp
 	result.Timestamp = metav1.Timestamp{Seconds: ruleResponse.Stats().Timestamp()}
+	pss := ruleResponse.PodSecurityChecks()
+	if pss != nil {
+		var controls []string
+		for _, check := range pss.Checks {
+			if !check.CheckResult.Allowed {
+				controls = append(controls, check.ID)
+			}
+		}
+		if len(controls) > 0 {
+			sort.Strings(controls)
```

Comment:
any specific reason to do a sort operation here?

Reply:
this part of code is present [here](https://github.com/kyverno/kyverno/blob/d737138ad51f71e5b7511ac53d01705fccc0bedc/pkg/utils/report/results.go#L114-L130), i just shifted that logic to this file

Reply:
the main reason here is just make it more readable, by sorting it alphabetically like this -
 `controls: allowPrivilegeEscalation,capabilities_restricted,runAsNonRoot,seccompProfile_restricted`